frontend/package.json

```diff
@@ -35,8 +35,11 @@
   },
   "type": "module",
   "dependencies": {
+    "@rails/activestorage": "^7.0.4-3",
     "@rgossiaux/svelte-headlessui": "^1.0.2",
     "@sveltejs/adapter-node": "^1.1.4",
+    "@types/rails__activestorage": "^7.0.1",
+    "antd": "^5.4.0",
```

Comment:
There's no need to import whole `andt` library (Which is quite heavy). You could just copy type if needed.